src/app/models/timeslot.rb

```diff
@@ -25,7 +25,7 @@ def initialize(start, stop)
     end
 
     def to_s(with_day: false)
-      "#{start_day + ' ' if with_day}#{start_time} - #{end_time}"
+      "#{start_day + ' ' if with_day}#{start_time} – #{end_time}"
```

Comment:
Heh, yeah, I may be just a little persnickety.

(And because that gets used in console output, something clearer like `&ndash;` isn't an option. Ruby doesn’t have named Unicode escape sequences, right? Just hex?)